lib/Timber.php

```diff
@@ -173,6 +184,21 @@ class_alias( 'Timber\Timber', 'Timber' );
 		}
 	}
 
+	/**
+	 * Initialize a single IntegrationInterface instance.
+	 *
+	 * @internal
+	 */
+	protected static function init_integration(
+		Integration\IntegrationInterface $integration
+	) : void {
+		if ($integration->should_init()) {
+			$integration->init();
+		} else {
+			echo 'DID NOT INIT ' . get_class($integration);
```

Comment:
I guess we shouldn’t leave a stray echo here?

Reply:
Removed!